jangjuseong/app.js

```diff
@@ -50,14 +50,148 @@ app.post('/users/signup', async (req, res) => {
         ?, 
         ?, 
         ?
-        );
+      );
     `,
     [name, email, password, profileImage]
   );
 
   res.status(201).json({ message: 'userCreated' });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, postImgUrl, userId } = req.body;
+
+  await mysqlDataSource.query(
+    `INSERT INTO posts (
+      title,
+      content,
+      post_image_url,
+      user_id
+      )
+      VALUES (
+        ?,
+        ?,
+        ?,
+        ?
+      );
+        `,
+    [title, content, postImgUrl, userId]
+  );
+
+  res.status(201).json({ message: 'postCreated' });
+});
+
+app.get('/posts', async (req, res) => {
+  const posts = await mysqlDataSource.query(
+    `SELECT
+      u.id AS userId,
+      u.profile_image AS userProfileImage,
+      p.id AS postingId,
+      p.post_image_url AS postingImageUrl,
+      p.content AS postingContent
+    FROM posts p
+    INNER JOIN users u ON u.id = p.user_id;
+    `
+  );
+
+  res.status(200).json({ posts });
+});
+
+app.get('/posts/user/:userId', async (req, res) => {
```

Comment:
이미 `:userId`라는 path parameter를 사용하기 때문에 posts 뒤의 /user path는 명시하지 않아도 될 것 같습니다.
```suggestion
app.get('/posts/:userId', async (req, res) => {
```